presto-orc/src/main/java/com/facebook/presto/orc/reader/ColumnReader.java

```diff
@@ -149,7 +156,7 @@ public int getResultSizeInBytes()
         throw new UnsupportedOperationException("Variable width streams must implement getResultSizeInBytes()");
     }
 
-    public void compactQualifyingSet(int[] surviving, int numSurviving)
+    protected void compactQualifyingSet(int[] surviving, int numSurviving)
```

Comment:
This looks like something we could use the template pattern for since it seems to be used in the same way for all concrete implementations.